### PR TITLE
ADMETICA: curation pipeline description

### PR DESCRIPTION
**Pull request by StLeonidas**
Added a description for data curation procedures

### CODE REVIEW

File: Descriptions/Curation.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+For model development, we explore the experimental background from which the data was obtained, and for prediction, we strongly recommend using the applicability domain of developed models to ensure if they are applicable to studied biochemical problems.
+
+## Common pipeline for data from different sources
+
+The preliminary curation process guarantees that the data from all used sources is compatible with each other and standardized for further model development. We focus on the following steps:
+1. Removing data that is not associated with both structural data and property/activity value.
+2. Basically, structural information and a value or binary result are sufficient for modeling, but additional fields of data points are preferable to handle when present:
+- Source type - a patent, a manuscript, a commercial database with assays - these categories can provide valuable information as they usually study different chemical classes, and the quality and volume of data can vary significantly.
```
Comment (Aleksashka11) [inline Descriptions/Curation.md:11]: Don't divide part of the sentence using '-'. Just:
```
Source type. 
A patent, manuscript, commercial database with assays. 
These sources offer valuable information, typically covering diverse chemical classes, with significant variations in data quality and volume.
```
So the whole structure will be just:
```
Field
What is included
Why we actually need it
```

Comment (Aleksashka11) [reply Descriptions/Curation.md:11]: Same for all remaining points

File: Descriptions/Curation.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- Value units - there is no hope to get accurate results with different units, fortunately they could be converted in most cases - like mM/L vs mg/L.
+- Endpoint type - different endpoints are difficult to mix for a single model development, properties are usually measured as a single endpoint, activities not  - Inh%, IC50, Ki are not even equal endpoints. Experimental conditions are not always present but play a crucial role, values obtained under different experimental conditions may interfere significantly, for example, solubility measured at 20 C or 35 C is different solubility. It is worth knowing and making different assumptions when a dataset is composed of data obtained in different experiments. Depending on the specific task, this field is just useful or crucial.
```
Comment (Aleksashka11) [inline Descriptions/Curation.md:13]: Somewhere '-', somewhere '.'. Let's improve it
And some of the sentences also can be split into several. Do not overload a sentence. For instance:
`Experimental conditions are not always present but play a crucial role, values obtained under different experimental conditions may interfere significantly, for example, solubility measured at 20 C or 35 C is different solubility.`

Comment (Aleksashka11) [reply Descriptions/Curation.md:13]: `
Experimental conditions, while not consistently present, play a crucial role. Values obtained under different conditions can significantly interfere. For instance, solubility measured at 20°C may differ from solubility measured at 35°C.`

File: Descriptions/Curation.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    >In well-known databases, such fields are standardized:
```
Comment (Aleksashka11) [inline Descriptions/Curation.md:14]: What for this '>' here?

File: Descriptions/Curation.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+in PubChem, this information is provided in the following fields - “Panel Name” or “Panel Target”. In ChEMBL, this information is provided in the following fields - “Assay Description” or “Description”, “Assay Organism”, “Assay Cell Type”, “Target ChEMBL ID”, “Target Name”, “Target Organism”. We DO NOT extract this information from manuscripts. Examples:
```
Comment (Aleksashka11) [inline Descriptions/Curation.md:15]: Brrr, don't like repetitions. This information is provided in the following fields twice.
```
Varying on the database, you can find this information in the following fields:

- PubChem: "Panel Name", "Panel Target"
- Chembl: etc
```
What I propose is more structured and easier to read

Comment (Aleksashka11) [reply Descriptions/Curation.md:15]: You just see Database and Fields

File: Descriptions/Curation.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+For antiviral activity, the “Assay description” may contain “Antiviral activity against HIV-1 in CEM cells” and “Inhibition of purified recombinant HIV-1 protease in a fluorogenic assay,” which means that the first experiment was in vivo and the second in vitro. Those two should never be mixed even if the studied endpoint is IC50 or Ki.
```
Comment (Aleksashka11) [inline Descriptions/Curation.md:16]: Simplify. When a sentence takes 2 and more rows, it is HARD to read and understand. Especially when we have quite a big flow

File: Descriptions/Curation.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+	"Eu ", "Gd ", "Tb ", "Dy ", "Ho ", "Er ", "Tm ",
+	"Yb ", "Lu ", "Hf ", "Ta ", "W  ", "Re ", "Os ",
+	"Ir ", "Pt ", "Au ", "Hg ", "Tl ", "Pb ", "Bi ",
+	"Po ", "At ", "Rn ", "Fr ", "Ra ", "Ac ", "Th ",
+	"Pa ", "U  ", "Np ", "Pu ", "Am ", "Cm ", "Bk ",
+	"Cf ", "Es ", "Fm ", "Md ", "No ", "Lr ", "Rf ",
+	"Db ", "Sg ", "Bh ", "Hs ", "Mt ", "Ds ", "Rg ",
+	"Cn ", "Nh ", "Fl ", "Mc ", "Lv ", "Ts ", "Og "
+3. Taking the major component: The desired property usually depends on the single structure. Entries in the dataset representing mixtures and salts contain more than one structural component. This can significantly affect generated descriptors/fingerprints - mixtures and single molecules will have different sets of features.
+>Depending on the task, it is not always relevant for salts (2 fragments, one of them is the ion of one atom). Salts should be handled by taking the major fragment and following neutralization (removing of charge) for the majority of molecular target associated models (like hERG) and not for properties.
```
Comment (Aleksashka11) [inline Descriptions/Curation.md:46]: Again this '>' (

File: Descriptions/Curation.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+	"Ba ", "La ", "Ce ", "Pr ", "Nd ", "Pm ", "Sm ",
+	"Eu ", "Gd ", "Tb ", "Dy ", "Ho ", "Er ", "Tm ",
+	"Yb ", "Lu ", "Hf ", "Ta ", "W  ", "Re ", "Os ",
+	"Ir ", "Pt ", "Au ", "Hg ", "Tl ", "Pb ", "Bi ",
+	"Po ", "At ", "Rn ", "Fr ", "Ra ", "Ac ", "Th ",
+	"Pa ", "U  ", "Np ", "Pu ", "Am ", "Cm ", "Bk ",
+	"Cf ", "Es ", "Fm ", "Md ", "No ", "Lr ", "Rf ",
+	"Db ", "Sg ", "Bh ", "Hs ", "Mt ", "Ds ", "Rg ",
+	"Cn ", "Nh ", "Fl ", "Mc ", "Lv ", "Ts ", "Og "
+3. Taking the major component: The desired property usually depends on the single structure. Entries in the dataset representing mixtures and salts contain more than one structural component. This can significantly affect generated descriptors/fingerprints - mixtures and single molecules will have different sets of features.
```
Comment (Aleksashka11) [inline Descriptions/Curation.md:45]: Please, decide which punctuation marks you use for separation)))
It is a mystery for me right now) I've already seen dots, hyphens and here is a colon (before you used hyphen)

File: Descriptions/Curation.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+Another major stage of curation deals with experimental numerical data. For this, we focus on the following:
+1. Removal of incorrectly annotated data/irrelevant data: We remove such data points if they are not specific to the model. For example, if developing a model for SARS-CoV-2 inhibitors, we exclude all entries for SARS-CoV, MERS, or SARS-CoV-2 mutants.
+2. Duplicates handling: Mixed datasets often contain several property/activity values for a single structure. We explore the distribution of these entries and exclude outliers. We usually take the median, and for concentration values, the harmonic mean or median of logarithmic transformed values are also accepted.
+>For modeling tasks, duplicates are not just equal structures. We compare and search for duplicates of descriptors/fingerprints vectors.
+3. Activity cliffs handling: The majority of machine learning approaches use smooth functions as a model. The last are fitted to capture the data, although structures highly similar to others but with a high difference in property/activity can give a strong bias to parameter estimates. We exclude them when there is just a single activity cliff in the subgroup.
```
Comment (Aleksashka11) [inline Descriptions/Curation.md:61]: We decided that it is hard to do automatically? Should we really include this step?

File: Descriptions/Curation.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+4. Normalization of values obtained in different testing systems: When property/activity values are obtained under different experimental conditions, we try to make a transformation considering differences in one of the experimental conditions.
```
Comment (Aleksashka11) [inline Descriptions/Curation.md:62]: Same is here

Comment (Aleksashka11) [reply Descriptions/Curation.md:62]: I just don't think that we need to specify it, especially when we don't do this

**Review by Aleksashka11 — CHANGES_REQUESTED** [review]

File: Descriptions/Curation.md (hunk 1 above)
Comment (StLeonidas) [reply Descriptions/Curation.md:11]: +

**Pull request merged**